connect/runtime/src/test/java/org/apache/kafka/connect/integration/TaskHandle.java

```diff
@@ -55,14 +58,54 @@ public void record() {
         connectorHandle.record();
     }
 
+    /**
+     * Record arrival of a batch of messages at the task.
```

Comment:
should we say that this records arrival of a batch at the task _and the connector_?